frontend/apps/crates/entry/user/src/register/pages/step_2/dom.rs

```diff
@@ -52,22 +55,46 @@ impl Step2Page {
                         dom.property_signal("error", state.language_error.signal())
                     })
                 ),
-                SimpleSelect::render_mixin(
-                    SimpleSelect::new(
-                        Some(STR_PERSONA_LABEL),
-                        Some(STR_PERSONA_PLACEHOLDER),
-                        None,
-                        STR_PERSONA_OPTIONS.to_vec(),
-                        clone!(state => move |value| {
-                            *state.persona.borrow_mut() = value.map(|x| vec![x.to_string()]);
-                            state.evaluate_persona_error();
+                html!("input-select", {
```

Comment:
This is duplicated logic - I think a good solution is to create a MultiSelect component like the SimpleSelect.